README.md

```diff
@@ -1,27 +1,49 @@
-# epm
-
-Ethereum Package Managment
+# EPM
 
 [![Join the chat at https://gitter.im/ethpm/Lobby](https://badges.gitter.im/ethpm/Lobby.svg)](https://gitter.im/ethpm/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-# Goals
+Ethereum Packaging Specification
```

Comment:
I want to stress that this is for more than Ethereum. It is for the EVM. We can go with either or after the change from Ethereum to EVM is made.